### PR TITLE
Last upgrade to CMSSW_10_3_0 was not fully successful (git issues)

### PR DESCRIPTION
**Pull request by giovanni-mocellin**
Some files from the local repository were not updated and was kept the original CMSSW version of them. Now everything is as it should be...

### CODE REVIEW

File: Geometry/MuonCommonData/plugins/DDMuonAngular.cc [hunk 1]
```diff
@@ -34,22 +30,19 @@ void DDMuonAngular::initialize(const DDNumericArguments & nArgs,
   n           = int (nArgs["n"]);
   startCopyNo = int (nArgs["startCopyNo"]);
   incrCopyNo  = int (nArgs["incrCopyNo"]);
-#ifdef EDM_ML_DEBUG
```
Comment (jshlee) [inline Geometry/MuonCommonData/plugins/DDMuonAngular.cc:37]: u can just compile in debug mode.
its not good to change files in other packages if possible.
u will end up with very large number of files to compile

File: Geometry/GEMGeometry/src/GEMEtaPartition.cc [hunk 2]
```diff
@@ -53,7 +53,7 @@ GEMEtaPartition::nstrips() const
 LocalPoint
```
Comment (jshlee) [inline Geometry/GEMGeometry/src/GEMEtaPartition.cc:53]: dont change this file

File: Geometry/GEMGeometry/python/gemGeometryCustoms.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
 import FWCore.ParameterSet.Config as cms
```
Comment (jshlee) [inline Geometry/GEMGeometry/python/gemGeometryCustoms.py:1]: dont add changes that does nothing

File: Geometry/GEMGeometry/interface/GEMEtaPartition.h [hunk 4]
```diff
@@ -33,16 +33,16 @@ class GEMEtaPartition : public GeomDet
   int nstrips() const;
```
Comment (jshlee) [inline Geometry/GEMGeometry/interface/GEMEtaPartition.h:33]: dont change this file

**Comment by giovanni-mocellin** [comment]
I've not changed any of those files, they were coming from the package we had in the previous version of CMSSW for the QC8 code (we were always running with those). When pushing today I tried not to add those to the new version of the code and to use the ones provided by CMSSW_10_3_0, but I got some errors, so I had to copy also those... Now everything runs fine.

**Comment by jshlee** [comment]
dont copy! always use official when possible
u are using old versions that will produce errors

**Comment by giovanni-mocellin** [comment]
I tried, but it did not compile without those files. A lot of errors are appearing. With those files, it works ok

**Comment by jshlee** [comment]
you are doing something wrong
the code is official cmssw has been tested and the one that is to be used.

**Comment by giovanni-mocellin** [comment]
You are right. On Monday, I will try to understand why it is working with the modules we have always been using and not with the new ones in the official version, and I will try to make it work with the official new files.

**Comment by jshlee** [comment]
looking at these files there is no issue
it just looks like git merge didnt work because the gem qc8 code is old.
and the merge conflicts are causing your compile problems

**Pull request closed without merging**